lua/framework/physics/lua_material.h

```diff
@@ -106,9 +106,9 @@ int MatAddProperty(lua_State* L);
  * will be converted to 0 to N-1. This method can be used to set mutligroup
  * cross-sections or sources.
  * \n
- * SIMPLEXS0\n
- * Makes a simple material with no transfer matrix just \f$\sigma_t \f$. Expects
- * two values: \n
+ * PURE_ABSORBER\n
+ * Makes a simple material with no transfer matrix and \f$\sigma_t \f$ =
+ * \f$\sigma_a \f$. Expects two values: \n
```

Comment:
```suggestion
 * Makes a simple material with no transfer matrix and \f$\sigma_t =
 * \sigma_a \f$. Expects two values: \n
```
Any reason for the `=` to not be in math font?